### PR TITLE
Use latest version of workload_launch for update.

### PR DESCRIPTION
**Pull request by sathlan**
This improve the comptatibility with different public network
setup (vlan segmentation for instance) and pave the way to get the
continuous control plane testing.

-------

As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

**Comment by raukadah** [comment]
recheck Ensure that user is logged crc failure on bm job

File: roles/update/templates/workload_launch.sh.j2 [hunk 1]
```diff
@@ -394,9 +499,18 @@ if [[ "${MODE}" == "cleanup" ]]; then
 fi
 
 if [[ "${MODE}" == "sanity" ]]; then
+    trap cleanup_on_exit EXIT
```
Comment (ciecierski) [inline roles/update/templates/workload_launch.sh.j2:502]: Why trap command here and next mode too?

**Review by ciecierski — CHANGES_REQUESTED** [review]
Is `trap `required?

File: roles/update/templates/workload_launch.sh.j2 (hunk 1 above)
Comment (sathlan) [reply roles/update/templates/workload_launch.sh.j2:502]: Sanity and sanifyfast mode, are special modes where vm are created and then destroyed.  sanityfast just has lower timeout values.

trap EXIT is a nice bash signal extension to make sure the cleanup code run whatever happen during deployment, either an error or a normal exit.

The code then doesn't have to be cluttered with conditional on every command to check if it succeed or not.  We are just certain that cleanup will try to run no matter what.

Comment (ciecierski) [reply roles/update/templates/workload_launch.sh.j2:502]: thank you for answer, makes sense to me

**Review by ciecierski — APPROVED** [review]

**Comment by cjeanner** [comment]
/approve

won't really break "us" afaik, so if it kills update, that's on them :).

**Pull request merged**